src/pytkdocs/loader.py

```diff
@@ -415,6 +433,49 @@ def get_property_documentation(self, node: ObjectNode) -> Attribute:
             source=source,
         )
 
+    def get_pydantic_field_documentation(self, node: ObjectNode) -> Attribute:
+        """
+        Get the documentation for a PyDantic Field
+
+        Arguments:
+            node: The node representing the Field and its parents.
+
+        Return:
+            The documented attribute object.
+        """
+        prop: ModelField = node.obj
```

Comment:
We removed the dependency, so you cannot import ModelField, therefore you have to use another annotation. No annotation is fine as well. Don't bother too much with mypy, add a `# type: ignore` if too hard to solve.